Gemfile

```diff
@@ -35,6 +35,7 @@ group :development do
   gem "puppet-module-win-dev-r#{minor_version}",       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "github_changelog_generator",                    require: false, git: 'https://github.com/skywinder/github-changelog-generator', ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018' if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')
   gem "puppet-strings",                                require: false
+  gem "webmock",                                       require: false
```

Comment:
Same as the rubocop.yml .  If you need to add this to the .sync.yml so pdk is aware of this development gem and won’t revert your change when the pdk community template is updated for this module in the future.

Let me know if you need help with this.